### ISSUE TITLE
macos 13 下 无法连接设备

### ISSUE THREAD

**Issue by Yutao-Jin** [RT-Thread/mpy-snippets#62]
显示no device found

**Comment by crisalgr** [comment]
did you manage to solve the problem? I'm using the VSCode extension 'RT-Thread MicroPython' on a macOS 14, with an M2 procesor and have the same issue

**Comment by Yutao-Jin** [comment]
yes try installing the previous version of the rt thread, I think it was 1.0.7or 1.0.8, one of these versions will work
